ChatTTS/model/gpt.py

```
@@ -1,301 +1,620 @@
+        logits_processors: Tuple[
+            Callable[[torch.LongTensor, torch.FloatTensor], torch.FloatTensor]
+        ] = (),
         infer_text=False,
         return_attn=False,
         return_hidden=False,
         stream=False,
+        show_tqdm=True,
+        ensure_non_empty=True,
+        stream_batch=24,
+        manual_seed: Optional[int] = None,
+        context=Context(),
```

Ruff (0.14.6)

Do not perform function call `Context` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)